### PR TITLE
chore: moved the top tabs from editor to extensions (Breaking change)

### PR DESCRIPTION
**Pull request by wilau2**
- Ran an import cleanup in the project since I've seen many of them. 
- Deleted a duplicated `classnames` file
- Move `Tags` an `Button` components from `@extension` to `@editor`
- Editor now exposes the 3 components (editorPreview, desktopPreview, mobilePreview)

This would be a breaking change:
From demo import `StandardEmailEditor` from `@extension` instead of `emailEditor` from `@editor`

TODO: 
- ~I need to update the readme.md will do it later after diner~ ✅

### CODE REVIEW

File: demo/src/pages/Editor/index.tsx [hunk 1]
```diff
@@ -498,26 +496,4 @@ export default function Editor() {
   );
 }
 
-function replaceStandardBlockToAdvancedBlock(blockData: IBlockData) {
```
Comment (wilau2) [inline demo/src/pages/Editor/index.tsx:501]: this was not used but I could remove the change if you want

File: demo/src/pages/Editor/index.tsx [hunk 2]
```diff
@@ -485,7 +483,7 @@ export default function Editor() {
                 compact={!smallScene}
                 categories={defaultCategories}
               >
-                <EmailEditor />
```
Comment (wilau2) [inline demo/src/pages/Editor/index.tsx:488]: This is the breaking change here

File: packages/easy-email-editor/src/components/EditEmailPreview/index.tsx [hunk 3]
```diff
@@ -9,6 +9,7 @@ import { DATA_ATTRIBUTE_DROP_CONTAINER, SYNC_SCROLL_ELEMENT_CLASS_NAME } from '@
 import { classnames } from '@/utils/classnames';
 import { ActiveTabKeys } from '@/components/Provider/BlocksProvider';
 import { useActiveTab } from '@/hooks/useActiveTab';
+import '@/assets/font/iconfont.css';
```
Comment (wilau2) [inline packages/easy-email-editor/src/components/EditEmailPreview/index.tsx:12]: I needed to move the font import here, I could not make it work from `@extension`

File: packages/easy-email-editor/src/index.tsx [hunk 4]
```diff
@@ -1,14 +1,9 @@
...
-
-export { EmailEditor } from './components/EmailEditor';
-
-// exposing more granular components
-export { EditEmailPreview } from'./components/EmailEditor/components/EditEmailPreview';
-export { MobileEmailPreview } from'./components/EmailEditor/components/MobileEmailPreview';
-export { DesktopEmailPreview } from'./components/EmailEditor/components/DesktopEmailPreview';
+export { EditEmailPreview } from './components/EditEmailPreview';
+export { MobileEmailPreview } from './components/MobileEmailPreview';
+export { DesktopEmailPreview } from './components/DesktopEmailPreview';
```
Comment (wilau2) [inline packages/easy-email-editor/src/index.tsx:7]: The breaking change is well seen here.

File: packages/easy-email-extensions/src/utils/getPreviewClassName.ts [hunk 5]
```diff
@@ -1,4 +1,4 @@
-import { classnames } from '@extensions/AttributePanel/utils/classnames';
```
Comment (wilau2) [inline packages/easy-email-extensions/src/utils/getPreviewClassName.ts:1]: removed the dup classname function to use utils instead

File: packages/easy-email-editor/src/utils/index.ts [hunk 6]
```diff
@@ -9,4 +9,4 @@ export { scrollBlockEleIntoView } from './scrollBlockEleIntoView';
 export { isTextBlock } from './isTextBlock';
 export { MergeTagBadge } from './MergeTagBadge';
 export { getContentEditableClassName } from './getContentEditableClassName';
-export { EventManager } from './EventManager';
\ No newline at end of file
+export { EventManager, EventType } from './EventManager';
```
Comment (wilau2) [inline packages/easy-email-editor/src/utils/index.ts:12]: added this export here

**Comment by m-Ryan** [comment]
Great job, we'll be done with the review this week.

**Review by m-Ryan — CHANGES_REQUESTED** [review]
The PR looks fine, but it looks like there will be a lot of conflicts with another PR.https://github.com/zalify/easy-email/pull/230
Most of the changes are formatting, maybe it can be solved by configuring eslint.

**Comment by wilau2** [comment]
closing this one for now since i'm running out of time and it was a breaking change refactor. Might reopen it later if I need it.

**Pull request closed without merging**